dask/array/rechunk.py

```diff
@@ -387,6 +388,23 @@ def rechunk(
         raise NotImplementedError(f"Unknown rechunking method '{method}'")
 
 
+def _choose_rechunk_method(old_chunks, new_chunks, threshold=None):
+    if method := config.get("array.rechunk.method", None) is not None:
```

Comment:
This is a little difficult to reason about. Maybe something like this instead?

```suggestion
    if config.get("array.rechunk.method", None) is not None:
```

Reply:
This is just new Python syntax or am I missing something?

Reply:
Yeah, sorry I should have made it clear that this was meant as a minor comment. The walrus operator hasn't made it into my day-to-day toolbox yet, so I always have to pause and look up what it does again. @hendrikmakait what you have here is fine.

Reply:
For anybody else who stumbles across this and wonders about the walru (`:=`) operator: https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Reply:
i feel you James. I do a double take every time even when I've written the code! 🙈